modules/bibupload/lib/bibupload.py

```diff
@@ -1212,27 +1212,33 @@ def check_record_doi_is_unique(rec_id, record):
     Return (boolean, msg) where 'boolean' would be True if the DOI is
     unique.
     """
-    record_dois = record_extract_dois(record)
-    if record_dois:
-        matching_recids = set()
-        for record_doi in record_dois:
-            possible_recid = find_record_from_doi(record_doi)
-            if possible_recid:
-                matching_recids.add(possible_recid)
-        if len(matching_recids) > 1:
-            # Oops, this record refers to DOI existing in multiple records.
-            msg = "   Failed: Multiple records found in the" \
+    try:
+        from invenio.config import CFG_BIBUPLOAD_DOI_CHECKING
```

Comment:
I would move the import together with the others, https://github.com/inveniosoftware/invenio/blob/legacy/modules/bibupload/lib/bibupload.py#L41, and probably you don't need the `try: ... except: ...` block as it's going to have a default value from `invenio.conf`